include/revng/ADT/STLExtras.h

```diff
@@ -339,3 +339,26 @@ replaceAll(std::string &Input, const std::string &From, const std::string &To) {
     Start += To.length();
   }
 }
+
```

Comment:
Typo in commit message `s/constepxr/constexpr/`

Reply:
Fixed: https://github.com/revng/revng/pull/241/commits/7992f1ccb944148fb0d84ff480a543d974d71bd8